### PR TITLE
Update featured stories

### LINKED ISSUE #1451: Add images to the featured story section of the Jakarta EE homepage via newsroom
Currently we are not able to add images in the newsroom to be featured on the homepage's featured story section. Adding images to that section can make the stories more visible to have a greater impact.

### PR DESCRIPTION
**Pull request by ericpoirier**
Fixes #1451 

Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

**Comment by ericpoirier** [comment]
We have the +1 from @shabnammayel

File: js/main.js [hunk 1]
```diff
@@ -44,14 +44,27 @@ document.addEventListener("DOMContentLoaded", function(event) {
...
+            autoplaySpeed: 2000,
+            loop:true,
+        });
+      }
+      
+      $(window).on("load", function() {
+          solsticeSliderHome();
+      });
+      
+      $("body").on("shown.ef.featured_story", function(e) {
```
Comment (chrisguindon) [inline js/main.js]: What is the purpose of this? 

I feel this we are doing something wrong when I see owl.trigger('destroy.owl.carousel'); then  owl.trigger('refresh.owl.carousel');

Comment (ericpoirier) [reply js/main.js]: its to make sure the carousel is loaded after the content is added. This is also something we are doing on ECDtools

Comment (chrisguindon) [reply js/main.js]: I feel this is the way we should do it:
https://www.jq22.com/demo/OwlCarousel2/demos/ajax.html

There is an event that you can trigger for inserting content.

Comment (ericpoirier) [reply js/main.js]: I'll give it a try.

Comment (ericpoirier) [reply js/main.js]: turns out I was using the same class for 2 different carousel on the page which was causing conflicts.

I update the class name for the Featured Story slider and removed the unnecessary code and both sliders are working well.

File: js/main.js [hunk 2]
```diff
@@ -44,14 +44,23 @@ document.addEventListener("DOMContentLoaded", function(event) {
...
-          autoplay:true,
-          autoplayTimeout:6000,
-          autoplayHoverPause:true,
-          autoplaySpeed: 2000,
-          loop:true,
+      var owlSolsticeSlider = $('.solstice-slider');
+      owlSolsticeSlider.owlCarousel();
+
+      const solsticeSliderHome = () => {
+        var owlSolsticeFeaturedStorySlider = $('.solstice-featured-story-slider');
```
Comment (chrisguindon) [inline js/main.js:51]: We don't/shouldn't be using var here.

I still think that the solution that I provided makes more sense but this is better than the previous revision.

$(window).on("load", function() { is fired when the whole page has loaded, including all dependent resources such as stylesheets and images.
 
 
 The solution that I provided can insert the owl carrousel once we receive the data from the API.

Comment (ericpoirier) [reply js/main.js:51]: I tried using the solution you suggested and it didn't seem work. The content is being added to the page but Owl isn't updated to create the carousel.

Comment (chrisguindon) [reply js/main.js:51]: You are right, this looks like an outdated version of the docs.

**Review by chrisguindon — APPROVED** [review]

**Pull request merged**